requires-install.txt

```diff
@@ -7,12 +8,12 @@ dash-html-components==1.0.0
 dash-table==4.0.0
 
 # dash.testing
-pytest
+pytest<5.0.0
```

Comment:
temporary ⬇️  pytest version until the fix in flask is released https://github.com/pallets/flask/pull/3278

@rpkyle  the unit tests all passed in circleci now